src/components/MapView/mapTest.html

```diff
@@ -0,0 +1,40 @@
+
+//test html
+
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title> first leaflet</title>
+	<link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.1/dist/leaflet.css"/>
+
+    <style>
+        #MapView{
```

Comment:
this is CSS (styling for the component), move this to MapView.css (should be in the same MapView directory)